tests/bdd/features/volume/resize/test_resize_offline.py

```diff
@@ -238,9 +298,10 @@ def the_volume_is_receiving_io():
 def the_volume_is_unpublished():
     """the volume is unpublished."""
     # Ensure fio is still running till unpublish
-    time.sleep(1)
-    assert pytest.fio.poll() is None
-    unpublish_volume()
+    time.sleep(0.5)
+    if pytest.fio.poll() is None:
```

Comment:
What does poll None mean again, that it's still running?

Reply:
Yes, `None` means running.